src/Substitution.java

```diff
@@ -0,0 +1,24 @@
+
+/**
+*  A substitution is a MadLibElement that holds a prompt that can be replaced 
+*/
+public class Substitution implements MadLibElement {
+    public String prompt, text;
+
+    public Substitution(String prompt) {
+        this.prompt = prompt;
+    }
+
+    public String getPrompt() {
+        return prompt;
+    }
+
+    @Override
+    public String getText() {
+        return this.text; 
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+}
```

Comment:
Good job following through on this structure I suggested throughout the code. You took it and ran with it!